core/templates/domain/opportunity/exploration-opportunity-summary.model.spec.ts

```diff
@@ -83,6 +86,56 @@ describe('Exploration opportunity summary model', () => {
         .toEqual(15);
     });
 
+    it('should return a correct translations in review counts', () => {
+      let explorationOpportunitySummary = (
```

Comment:
Can move this initialization to the `beforeEach` and update corresponding tests. Also can make a `const`.

Reply:
Done kept it as `let` as the variable is being changed in each test.